### DISCUSSION TITLE
如何让稍后读支持读取本地缓存

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2146] [服务]
# 重要提示

> ### 嗨，简悦全新的 [教程中心](https://www.yuque.com/kenshin/simpread) 已上线，**下面的内容并不一定是最新的**，建议 [访问这里](https://www.yuque.com/kenshin/simpread/wkswh7) 查看最新版教程。

***

> 当你使用 Pocket 类产品的一个很重要原因：它可以缓存稍后读的页面，也就是当原页面不存在时，在 Pocket 里面仍可以看到。

> **[简悦 2.2.0](http://ksria.com/simpread/welcome/version_2.2.0.html)** 发布后，就可以很容易的实现这种方式，且有更强的定制化。

## 优势

> 相比 Pocket 之类的产品，简悦的优势在于：**本地数据**，也就是说，通过这种方式得到的数据是在你的本地，你可以通过 **坚果云 或 Dropbox 同步这些数据到任意设备上**，同时还支持类似 DEVONThinks 这样的检索工具辅助检索。

> 相比 Pocket 这种 **在线服务来说，这就是最大优势**。

## 前提

> 简悦因为灵活的使用方式导致此功能并没有所谓的「开关」开启它，需要多种功能协同完成，但过程并不困难。

## 前期准备

1. 需要 [**简悦 · 高级账户**](https://simpread.pro/price.html)

### 配置自定义标题

> 设置为 `{{id}}{{un_title}}{{mode}}`，如下图所示

![image](https://user-images.githubusercontent.com/81074/119208921-c4a75380-bad6-11eb-88eb-62c70d1109cc.png)

> 关于自定义标题的更多说明 [请看这里](http://ksria.com/simpread/docs/#/%E5%AE%9A%E5%88%B6%E5%8C%96%E5%AF%BC%E5%87%BA?id=%e8%87%aa%e5%ae%9a%e4%b9%89%e5%af%bc%e5%87%ba%e6%a0%87%e9%a2%98)。

### 配置增强导出

> 详细说明请看教程 [如何配置增强导出](https://github.com/Kenshin/simpread/discussions/2958)，关于增强导出的更多说明 [请看这里](http://ksria.com/simpread/docs/#/Sync?id=%e5%af%bc%e5%87%ba%e6%9c%8d%e5%8a%a1)。

### 配置稍后读

> 选项页 → 高级设定 → 稍后读（**必须要开启下面的选项**）

![image](https://user-images.githubusercontent.com/81074/138629445-b05a3127-3d57-4075-adcc-35aab085bf95.png)

> 打开稍后读 → 左下角 → 打开选项页，并勾选下面的选项

![image](https://user-images.githubusercontent.com/81074/119209007-1059fd00-bad7-11eb-9a8e-051ce8eca24d.png)

> 选项页 → 高级设定 → 标注（**建议开启下面的选项**）_更新日期：2021年10月25日_

![image](https://user-images.githubusercontent.com/81074/138630288-380a5526-fb14-41af-8dc4-0fe9d3c72092.png)

### 配置 [自动化](http://ksria.com/simpread/docs/#/%E8%87%AA%E5%8A%A8%E5%8C%96)

> 加入一个新的自动化方案，简单的说：就是**当保存到稍后读后，执行导出 HTML 的功能**，如下图所示

![image](https://user-images.githubusercontent.com/81074/119209112-8eb69f00-bad7-11eb-92c0-16dd574325c2.png)

> 简单的引申下，你可以在导出 HTML 时，也导出到你常见的服务。（这个话题跟本教程无关，就不在引申讨论。

### 配置结束

> 截至到现在为止，全部的配置已经结束，接下来是如何使用。

## 使用

> 需要先进入阅读模式后，使用 快捷键 或 右键 加入稍后读后会有类似这样的提示

![2021-05-22_08-45-37 (1)](https://user-images.githubusercontent.com/81074/119209674-7eec8a00-bada-11eb-9461-919ff9ef8fdb.gif)

> 然后在 同步助手设置的 `output` 文件夹中会找到下面的文件

![image](https://user-images.githubusercontent.com/81074/119209644-63817f00-bada-11eb-9c2e-eaa144f3b70f.png)

> 下面这个操作全部的过程

![2021-05-22_08-51-38 (1)](https://user-images.githubusercontent.com/81074/119209787-4e592000-badb-11eb-8fb5-d84e6875720e.gif)

## 如何判断当前的稍后读是本地还是网络读取？

> 看下图，有两个标识。

![image](https://user-images.githubusercontent.com/81074/119209901-f7a01600-badb-11eb-860e-c0eb1cb8dfdc.png)

## 备注

> 上面的操作中我使用了 **HTML** 而非 **离线 HTML**（其实是一致的）

## 注意

> 这里有一个 Bug，影响不大，细节 [请看这里](https://github.com/Kenshin/simpread/discussions/3098)。

## 引申

> 上述操作只是 当加入稍后读后，产生本地环境，如果你想在产生标注时，也同样使用本地缓存的话，请看 [这个配置](https://github.com/Kenshin/simpread/discussions/2220)

## 参考知识点

> 上述操作都已经写到了文档里面，有感兴趣的用户，可以看下。

- 简悦 · 同步助手的 [增强导出功能](http://ksria.com/simpread/docs/#/Sync?id=%e5%af%bc%e5%87%ba%e6%9c%8d%e5%8a%a1)
- [自定义标题](http://ksria.com/simpread/docs/#/%E5%AE%9A%E5%88%B6%E5%8C%96%E5%AF%BC%E5%87%BA?id=%e8%87%aa%e5%ae%9a%e4%b9%89%e5%af%bc%e5%87%ba%e6%a0%87%e9%a2%98)
- [自动化](http://ksria.com/simpread/docs/#/%E8%87%AA%E5%8A%A8%E5%8C%96)

**Comment by hezhefly** [comment]
是我来早了，都是404，大神慢慢写，不急

**Reply by Kenshin** [reply]
抱歉，是突然更新了，明天会尽快补上文档。

**Reply by Kenshin** [reply]
补一个需要提示的 Issues https://github.com/Kenshin/simpread/discussions/2146

**Reply by lemonleaves-Tang** [reply]
非常感谢，很好的解决方案。

**Comment by luc99hen** [comment]
”加入稍后读“自动化是一定要进入阅读模式才能触发吗？直接在原网页上加入稍后读好像是不行的。

**Reply by Kenshin** [reply]
> ”加入稍后读“自动化是一定要进入阅读模式才能触发吗？

是的

> 直接在原网页上加入稍后读好像是不行的。

接下来会考虑此方式，因为在未进入阅读模式前，无法确保获取的正文是否精准。

**Comment by HJCFLY** [comment]
我的不行哎

**Reply by Kenshin** [reply]
不行的意思是什么？右键菜单没有保存到稍后读吗？

如果是的话，去 **选项页 → 高级设定 → 右键菜单** 开启相关选项。

文档已经提到了，支持 快捷键 与 右键菜单。

**Comment by HJCFLY** [comment]
确实是这个问题，十分感谢！但是我在阅读模式直接加入稍后阅读后，是只有题目，显示如下。请问这个原因是为什么？我哪里又设置错了嘛？

**Reply by Kenshin** [reply]
没理解 **只有题目** 是什么意思？最好能录屏给我看下。

**Reply by maijianbang** [reply]
同样的问题，SSPAI的文章添加到稍后读之后只显示标题，什么内容都没显示

**Reply by Kenshin** [reply]
> 同样的问题，SSPAI的文章添加到稍后读之后只显示标题，什么内容都没显示

请问你配置了吗？如果没有安教程配置的话，并且也没有标注的话，当然只有标题了。

**Reply by maijianbang** [reply]
> > 同样的问题，SSPAI的文章添加到稍后读之后只显示标题，什么内容都没显示
> 
> 请问你配置了吗？如果没有安教程配置的话，并且也没有标注的话，当然只有标题了。

配置了，output目录下的离线html文件打开没问题，但是稍后读就只显示文章标题。而且就算不配置，稍后读也不应该只显示文章标题吧？

**Reply by Kenshin** [reply]
估计是配置造成的，在 Github 找到我的邮箱地址给我邮件，并附上配置文件，我帮忙你看看~

**Reply by myjeshurun** [reply]
我也出现同样的问题。怎么办？

**Reply by Kenshin** [reply]
@myjeshurun 

嗨， 你的问题已经彻底解决，解决方案请看 https://github.com/Kenshin/simpread/discussions/2973

@maijianbang

建议也按照  https://github.com/Kenshin/simpread/discussions/2973 的说明进行设置，看看能否解决你的问题。

**Comment by HJCFLY** [comment]
感谢大佬的耐心，过程录屏如附件。但是如下图，里面没内容。

**Reply by Kenshin** [reply]
不要通过邮件的形式回复我，在 Github 里面回复我，否则什么都看不到… 😂

**Reply by Kenshin** [reply]
@HJCFLY 

嗨，你的问题建议也按照 #2973 的说明进行设置，看看能否解决你的问题。

**Comment by ASIjacket** [comment]
按这个步骤设置以后，刷新重启自动化任务还在，但是执行添加到稍后读以后， 那个自动化任务就消失了  当然也没保存到本地，可能是什么原因呢？

**Reply by Kenshin** [reply]
> 那个自动化任务就消失了

是指在选项页 → 服务 里面也没有显示？

如果是的话，估计有可能是配置被覆盖了，你可以理解为 `多线程` 问题，虽然有保护措施，但确实在特殊情况下（隐性 Bug）会产生此问题。

你可以这样做：

请先设置自动化，然后重启浏览器后（其实原本不需要重启，但既然发生了类似问题，就确保万一）看看是否还存在，如果存在的话，你可以再试试稍后读的自动化方案是否也执行了。

**Reply by ASIjacket** [reply]
> > 那个自动化任务就消失了
> 
> 是指在选项页 → 服务 里面也没有显示？
> 
> 如果是的话，估计有可能是配置被覆盖了，你可以理解为 `多线程` 问题，虽然有保护措施，但确实在特殊情况下（隐性 Bug）会产生此问题。
> 
> 你可以这样做：
> 
> 请先设置自动化，然后重启浏览器后（其实原本不需要重启，但既然发生了类似问题，就确保万一）看看是否还存在，如果存在的话，你可以再试试稍后读的自动化方案是否也执行了。

对的，是指 选项-> 服务里的自动化都清空了。
本地配置覆盖掉了，粗略认为可能是自动同步存在延迟，其他的服务设置选项有时也会不立即生失效，坚果云有多次配置文件更改记录，我关闭自动同步以后，设置就没有问题了。

希望大大可否有一个首次使用的设置指引，依次设置好基础配置，我刚开始使用时总会遗漏掉一些设置，要在文档里来回翻，有些问题并没有明显提示要完整读过文档才能发现，效率确实很低。

**Reply by Kenshin** [reply]
> 希望大大可否有一个首次使用的设置指引，依次设置好基础配置，我刚开始使用时总会遗漏掉一些设置，要在文档里来回翻，有些问题并没有明显提示要完整读过文档才能发现，效率确实很低。

谢谢提示，简悦之前只有文档，现在有了教程，比如这个 Discussion 其实就是篇教程。
当然后续我会持续推出类似这样的教程，甚至于视频教程。（但无奈时间关系，所以只能慢慢来。

**Comment by lasyjobs** [comment]
这里好像缺一个如何挂载本地离线HTML文件的方法，我通过浏览器的阅读模式离线了文件想添加到稍后读里面。

**Reply by Kenshin** [reply]
嗨，你是指：

> 浏览器打开本地 HTML，然后再加入到稍后读？

**如果是的话，其实这个功能是存在的**，但因为一个 Bug 导致它无法使用。  😂

**Reply by lasyjobs** [reply]
我是想不通过浏览器，自己在稍后读里面打开本地的html文件，稍后读兼容HTML文件的管理工作。

**Reply by Kenshin** [reply]
> 我是想不通过浏览器，自己在稍后读里面打开本地的html文件，稍后读兼容HTML文件的管理工作。

emmm 目前没有这样的功能，可以考虑下。

**Reply by Kenshin** [reply]
引申下：

如果你只是纯粹的想「管理本地 HTML」的话，应该有很多工具，除非你需要对本地 HTML 进行标注等操作，才有意义。

**Reply by maijianbang** [reply]
> 引申下：
> 
> 如果你只是纯粹的想「管理本地 HTML」的话，应该有很多工具，除非你需要对本地 HTML 进行标注等操作，才有意义。

求推荐ios和pc端阅读和管理html的软件，找了好久没找到😀

**Reply by Kenshin** [reply]
> 求推荐ios和pc端阅读和管理html的软件，找了好久没找到😀

我在 Mac 上使用 DEVONThinks 管理全部的本地文件，很方便；除此之外的话，也可以试试 Zotero / tagLyst

简悦也在配合 DEVONThinks 来辅助我管理 https://github.com/Kenshin/simpread/discussions/2221

**Reply by maijianbang** [reply]
devonthinks可太贵了，ios端也用devonthinks？

**Reply by Kenshin** [reply]
> devonthinks可太贵了，ios端也用devonthinks？

我从不使用 iOS 看太多内容，我只把它当娱乐工具。所以，也不需要管理。 😂

**Reply by maijianbang** [reply]
> > devonthinks可太贵了，ios端也用devonthinks？
> 
> 我从不使用 iOS 看太多内容，我只把它当娱乐工具。所以，也不需要管理。 😂

期待简悦能读取本地html

**Reply by lasyjobs** [reply]
> > 引申下：
> > 如果你只是纯粹的想「管理本地 HTML」的话，应该有很多工具，除非你需要对本地 HTML 进行标注等操作，才有意义。
> 
> 求推荐ios和pc端阅读和管理html的软件，找了好久没找到😀
本地管理肯定会有标记的

**Reply by Kenshin** [reply]
@lasyjobs 

嗨，你的诉求已经记录了，不用特意开一个 Issues （我已经删除了这个重复的 Issues

**Reply by YB-Kang** [reply]
@Kenshin 楼上的这个建议很好，我也补充一个建议，tg bot实现了多网页批量保存到稍后读，希望也能批量导出到本地。

**Reply by Kenshin** [reply]
> @Kenshin 楼上的这个建议很好，我也补充一个建议，tg bot实现了多网页批量保存到稍后读，希望也能批量导出到本地。

通过简悦的稍后读批量导出到本地？

**Reply by YB-Kang** [reply]
@Kenshin 对。通过tg bot加入到稍后读不会自动导出到本地，我认为放本地比较安全

**Reply by Kenshin** [reply]
@KYBIN-9 

收到，会考虑的。

**Reply by YB-Kang** [reply]
@Kenshin 我的设想是这样的，我平时习惯用obsidian，比如说我们要做某个专题的学习，或者资料的整理，一个个打开网页然后导出我觉得不方便，所以我习惯打开多个网页，然后一键复制所有标签页的网页链接用，用 tg bot 加入到稍后读，如果可以同时一键导出的话，我可以方便用obsidian来整理和管理。

**Reply by Kenshin** [reply]
@KYBIN-9

okay 了解使用场景了。

**Reply by Leo-Mu** [reply]
@Kenshin 这个功能实现了吗？

**Reply by Kenshin** [reply]
@Leo-Mu 

哪个功能？

**Comment by googlefish** [comment]
按教程配置了，稍后读里也显示已本地保存成功，但是仍然提示“解析正文失败”。貌似没有去读本地的文件。
检查本地已有对应的html，且可以打开。另外这个{{id}}{{un_title}}{{mode}} 标题配置好象无效，本地保存的文件名没有id，把标题配置只改成“{{id}}，自动保存的文件名依然与”{{id}}{{un_title}}{{mode}}“是一样的。

**Reply by Kenshin** [reply]
emmm 因为给出的信息太少（当然，虽然说了是按照教程）我没法判断。

你可以把配置文件以及这个 discussions ID 发送到 `help######simpread.pro` 邮箱里面，我看下。

**Reply by Kenshin** [reply]
嗨，建议按照 #2973 的说明进行设置，看看能否解决你的问题。

**Reply by googlefish** [reply]
多谢了。发现是有些页面不可以。有些是可以按模板id+title生成本地文件的。
以下是有问题的链接：https://www.infoq.cn/article/kXCDJ0gMEE4cSv05XVNq

**Reply by Kenshin** [reply]
> 多谢了。发现是有些页面不可以。有些是可以按模板id+title生成本地文件的。 以下是有问题的链接：https://www.infoq.cn/article/kXCDJ0gMEE4cSv05XVNq

你是说这个链接无法通过加入稍后读自动化后生成 HTML？如果是的话，你是如何操作的？
另外，是否安装了 https://simpread.ksria.cn/plugins/details/tMGXrU1v0U 这个插件？

**Reply by googlefish** [reply]
访问上面这个链接时，我加入稍后读，可以自动化生成html（生成的文件名没有按照定制的标题规则生成，没有id，但内容是正常的），在稍后读里 的“阅读模式”时有问题，看不到内容，只有标题。提示“异步加载型页面，本次未能获取到内容元素。请将 infoq.cn 加入排除列表后，刷新页面并手动进入阅读模式尝试”，按提示操作后，问题依旧。
您说的插件没有装过，刚才装了一下，还是同样的问题。

**Reply by Kenshin** [reply]
> 生成的文件名没有按照定制的标题规则生成，没有id

这应该是 https://github.com/Kenshin/simpread/discussions/2973 造成的问题。

> 在稍后读里 的“阅读模式”时有问题，看不到内容

同样，你应该设置了 `{{id}}{{un_title}}{{mode}}` 的形式，当导出的 HTML 不含有 `id` 时，则是读不到这个文件的。

**Reply by googlefish** [reply]
尝试将“定制导出” 改成{{un_title}}{{mode}}后再执行，一切正常了。加入“稍后读”，在“稍后读”里可以正常阅读离线内容了。感谢~~

**Reply by Kenshin** [reply]
> 尝试将“定制导出” 改成{{un_title}}{{mode}}后再执行，一切正常了。加入“稍后读”，在“稍后读”里可以正常阅读离线内容了。感谢~~

哈哈，搞定就好。

虽然能解决你的问题，但还是不应该发生才对。（你原意的话有时间可以把配置文件发到 help#####simpread.pro 我看下。(并注明 discussions ID

**Comment by invisprints** [comment]
不知道可否支持将本地缓存导入到稍后读中？之前也遇到 https://github.com/Kenshin/simpread/discussions/2146#discussioncomment-1536500 类似的问题，手贱将部分文章从稍后读中删除了。他们的本地缓存还在，但发现无法导入进稍后读了😂

**Reply by Kenshin** [reply]
其实是可以的，因为我 2.2.0 Release 前的一个傻x 设计（之二），导致会出现错误，只能下个版本修复了。

**Comment by SuperCicada** [comment]
我想用稍后读里打开收藏助手保存在output文件夹里的html，可以实现吗？

**Reply by Kenshin** [reply]
嗨，当然可以了，看下图所示

![image](https://user-images.githubusercontent.com/81074/162658147-a8978951-286b-494e-a479-440da772e26c.png)

**Reply by SuperCicada** [reply]
emm, 我用稍后读导出至output文件夹时候, 根据知识库的设置操作, 同时产生html和markdown两个文件, 文件名称也用了模板, 这种的情况在稍后读里是如上图一样.   

但是我用收藏助手(simpread.pro/@ID)收藏的url, 也就是在output里产生的那个HTML文件, 在阅读列表里可以显示, 在稍后读里加载不出来.    (稍后读设置里已开启了优先加载本地的离线文件选项.)  

再一次但是, 我在阅读列表打开那篇收藏的网页, 右键-简悦-将当前页加入稍后读, 这样就可以在稍后读里加载了.  我想省掉这一步.

另外收藏助手收藏的今日头条的url,在阅读列表里总是解析失败.

**Reply by SuperCicada** [reply]
当我在(simpread.pro/@id)的阅读列表里打开文章, 再加入稍后读, 会在output里出现三个文件, 其中一个是收藏助手收藏时产生的html, 文件名是数字+文章标题.
另外两个文件是稍后读产生的html和markdown, 文件名用了跟上面不同的数字+格式(html/markdown)+文章标题. 
但是我并没有从稍后读导出至output文件夹, 就产生了这两个文件.
在新打开的网页(非阅读列表中的), 加入稍后读并不会产生这两个文件, 导出后才有.

**Reply by SuperCicada** [reply]
哦, 阅读列表是可以把三个文件都加载出来的. 

用收藏助手, 新收藏的网页, 文件名改成数字+html+文章标题,(像稍后读导出的一样), 稍后读也加载不出来, 

但阅读列表刷新可以重新加载出来, 文件名也更新了.

**Reply by Kenshin** [reply]
嗨，上述问题，其实需要配置：

1. [知识库](https://kb.simpread.pro/#/page/%E5%BB%BA%E7%AB%8B%E7%9F%A5%E8%AF%86%E5%BA%93)
2. [资料库](https://kb.simpread.pro/#/page/%E5%BB%BA%E7%AB%8B%E8%B5%84%E6%96%99%E5%BA%93)

就能解决了。

**Reply by SuperCicada** [reply]
之前就已经配置过知识库和资料库了, 刚才又看了一遍, 按步骤每一步都点了点, 没发现之前有遗漏和错误的地方.

**Reply by Kenshin** [reply]
那只能这样了...

请通过 https://github.com/kenshin/simpread/issues/new 新建一个 Issues，并详细描述（截图或视频）你的问题。

只是凭借文字描述并没有办法能精准找到你的问题。

另外，我总结了稍后读近期的一些问题汇总到 https://zhuanlan.zhihu.com/p/497108825 看看是否可以帮到你。

**Reply by SuperCicada** [reply]
谢谢磊神耐心回复, 知乎这篇问题汇总里的最后一部分, (本地html加入稍后读), 应该就是我需要的.

**Reply by Kenshin** [reply]
> 谢谢磊神耐心回复, 知乎这篇问题汇总里的最后一部分, (本地html加入稍后读), 应该就是我需要的.

哈哈，搞定回来说声，否则就新建个 issues 描述你的问题。

**Reply by SuperCicada** [reply]
我的问题和Discussion #2921 里的用户YangWaldon的是一模一样的

**Reply by Kenshin** [reply]
> 我的问题和Discussion #2921 里的用户YangWaldon的是一模一样的

然后呢？还是没有搞定？

**Reply by SuperCicada** [reply]
搞定啦!搞定啦! ~~ 太爽了

**Comment by yomizzz** [comment]
<img width="866" alt="截屏2022-04-15 下午8 44 07" src="https://user-images.githubusercontent.com/22812174/163572369-5d57557f-a0f3-429b-b8c5-835f1edacd9b.png">
我在Mac上添加一篇文章到稍后读，就是图片里的205，但是在稍后读里不显示。然后我在win电脑上添加文章，又会重复205这个序号。然后在mac上稍后读无法显示本地缓存文章，如图所示，很奇怪。大佬，帮忙看看是什么问题?

**Reply by yomizzz** [reply]
发现原因了，原来是因为我在output文件夹里放了其他文件夹导致的😂

**Reply by Kenshin** [reply]
哈哈，搞定就好，这个「问题」下个版本修复。

**Comment by HQ4AI** [comment]
您好，请问：按照教程执行后，导出的html没有执行到我在同步助手里设定的目录，而是保存到chrome的下载文件夹。
请问这个怎么解决？

**Reply by Kenshin** [reply]
设置增强导出，建议直接看 https://www.yuque.com/kenshin/simpread/wkswh7 这个教程。

如果仍有问题，不要在这里提问了，请提 Issues https://github.com/kenshin/simpread/issues/new

**Comment by kohyzsc123** [comment]
自动化设置了自动保存html和markdown，但是加入稍后读只执行了保存markdown是什么原因呢

**Reply by Kenshin** [reply]
详细说明你的问题，比如截图给我看看自动化的内容等。

留意下：

1. 进入阅读模式后才能使用此功能。
2. 触发自动化，仅支持快捷键 <kbd>d d</kbd> 或者右键菜单。

**Comment by GGarYY1997** [comment]
为什么本地md html 稍后读里不显示本地缓存标志
增强导出一定要4个全开吗

**Reply by Kenshin** [reply]
> 为什么本地md html 稍后读里不显示本地缓存标志

对比下你的配置跟教程 https://www.yuque.com/kenshin/simpread/wkswh7 的区别。

> 增强导出一定要4个全开吗

不需要，看你自身的使用需求...

**Reply by Kenshin** [reply]
请不要通过 Email 回复，请进入 https://github.com/Kenshin/simpread/discussions/2146#discussioncomment-6221581 回复。

**Reply by GGarYY1997** [reply]
![image](https://github.com/Kenshin/simpread/assets/136989982/31375741-2da1-427e-acbe-ab5294c15353)
![image](https://github.com/Kenshin/simpread/assets/136989982/9e6c93b1-1f10-47e9-b5df-8c36cbb213f3)
稍后读还是没有显示本地存储的图标

**Comment by a12591771** [comment]
我有很多已经离线的html，可以导入吗？

**Reply by Kenshin** [reply]
可以的，如果使用同步助手的话，可以使用 https://github.com/Kenshin/simpread/discussions/4625 将这些离线文件也生成阅读模式（进而可以加入稍后读）等操作。
